src/Umbraco.Web/Models/Mapping/ContentModelMapper.cs

```diff
@@ -142,9 +142,33 @@ private static void AfterMap(IContent content, ContentItemDisplay display, IData
                 TabsAndPropertiesResolver.AddListView(display, "content", dataTypeService);
             }
 
+            var currentDocumentType = contentTypeService.GetContentType(display.ContentTypeAlias);
+            var currentDocumentTypeName = currentDocumentType == null ? String.Empty : currentDocumentType.Name;
+            var docTypeValue = String.Empty;
+
+#if (DEBUG)

```

Comment:
Using #ifdef with Debug would mean this would only ever work for a custom build in Debug mode, which we never ship with so this would mean to make this work you'd have to be running a custom build of Umbraco in Debug. 

The debug flag in the Web.Config gets carried through in HttpContext.IsDebuggingEnabled